Framework/MDAlgorithms/src/CompactMD.cpp

```diff
@@ -118,7 +118,9 @@ void CompactMD::exec() {
 
   const size_t nDimensions = input_ws->getNumDims();
   std::vector<Mantid::coord_t> minVector;
+  minVector.reserve(nDimensions);
   std::vector<Mantid::coord_t> maxVector;
+  maxVector.reserve(nDimensions);
```

Comment:
I'd remove these reserves. I really doubt they help at all performance wise, instead they just add noise to the code.